versioned_docs/version-24.04/administration/centreon-ha/acceptance-guide.md

```diff
@@ -0,0 +1,712 @@
+
+First, check the constraints:
+
+```shell
+pcs constraint
+```
+
+The command should return this:
+
+```text
+Location Constraints:
+Ordering Constraints:
+Colocation Constraints:
+  centreon with ms_mysql-clone (score:INFINITY) (rsc-role:Started) (with-rsc-role:Master)
+  ms_mysql-clone with centreon (score:INFINITY) (rsc-role:Master) (with-rsc-role:Started)
+Ticket Constraints:
+```
+
+then check the database replication
+
+```bash
+/usr/share/centreon-ha/bin/mysql-check-status.sh
+```
+
+At this time, the cluster is in degraded mode with two passive nodes.
+In this particular case, it returns the following information because the ms_mysql resource is stopped on node 1:
+
+```text
+Connection SLAVE Status '@CENTRAL_NODE1_NAME@' [KO]
+Error reports:
+    ERROR 2002 (HY000): Can't connect to MySQL server on '@CENTRAL_NODE1_NAME@' (115)
+Impossible de se connecter au serveur '@CENTRAL_NODE1_NAME@'.
+Connection SLAVE Status '@CENTRAL_NODE2_NAME@' [OK]
+Slave Thread Status [KO]
+Error reports:
+    Skip check on '@CENTRAL_NODE1_NAME@'.
+    No slave (maybe because we cannot check a server).
+Position Status [SKIP]
+Error reports:
+    Skip because we can't identify a unique slave.
+```
+
+You must do a database synchronization from @CENTRAL_NODE2_NAME@ to @CENTRAL_NODE1_NAME@ by launching the "sync-bigdb" script on the **Slave node**.
+
+```shell
+/usr/share/centreon-ha/bin/mysql-sync-bigdb.sh
+```
+
+As for the previous execution of this script, check whether the LVM Snapshot is correctly deleted and the MySQL Slave restarted:
+
+```text
+Umount and Delete LVM snapshot
+  Logical volume "dbbackupdatadir" successfully removed.
+Start MySQL Slave
+OK
+Start Replication
+Id      User    Host    db      Command Time    State   Info    Progress
+5       centreon-repl   @CENTRAL_NODE2_NAME@:51850        NULL    Query   0       starting        show processlist  0.000
+6       centreon        localhost       centreon_storage        Sleep   0               NULL    0.000
+7       system user             NULL    Connect 0       Connecting to master    NULL    0.000
+8       system user             NULL    Slave_SQL       0       Slave has read all relay log; waiting for more updates    NULL    0.000
+```
+
+Database replication should now be OK. Check this.
+
+```shell
+/usr/share/centreon-ha/bin/mysql-check-status.sh
+```
+
+The result should be:
+
+```text
+Connection MASTER Status '@CENTRAL_NODE2_NAME@' [OK]
+Connection SLAVE Status '@CENTRAL_NODE1_NAME@' [OK]
+Slave Thread Status [OK]
+Position Status [OK]
+```
+
+Now, you can perform a failover to return to the initial situation.
+
+```shell
+pcs resource clear centreon
+```
+
+Do a cleanup to clear errors and restart the ms_mysql resource on @CENTRAL_NODE1_NAME@.
+
+```shell
+pcs resource cleanup
+```
+
+The situation has stabilized, and you can perform a failover by moving the **centreon** resource.
+
+```shell
+pcs resource move centreon
+```
```

Comment:
Not sure why we do this here. But if we do, we must clear the constraints afterwards...